runtime/bindings/python/CMakeLists.txt

```diff
@@ -191,7 +191,14 @@ install(
 )
 
 # Install tools into python_packages/iree_runtime/iree/runtime
-install(
+#
+# Our runtime/... directory is included by the root CMakeLists before the
+# tools/ directory which defines these targets, so we defer the install() to
+# the end of the root file. While deferred calls are generally fragile, this
+# install is purely a leaf feature (with no other calls depending on its
+# sequencing), so this use is okay.
+cmake_language(DEFER DIRECTORY "${IREE_SOURCE_DIR}"
```

Comment:
```
CMake Error at runtime/bindings/python/CMakeLists.txt:200 (cmake_language):
  Unknown CMake command "cmake_language".
```

https://cmake.org/cmake/help/latest/command/cmake_language.html#deferring-calls
> _New in version 3.19._

Let's update?
https://github.com/google/iree/blob/5051a6171d52ac8b4876561e136f88d231e0230a/CMakeLists.txt#L7 https://github.com/google/iree/blob/5051a6171d52ac8b4876561e136f88d231e0230a/build_tools/docker/base/Dockerfile#L59-L61

Reply:
I'm going to send a separate PR to update our CMake min version then rebase this change. We discussed updating CMake [here on Discord](https://discord.com/channels/689900678990135345/689906000043573354/975823645886775366)